sormas-ui/src/main/java/de/symeda/sormas/ui/task/TaskController.java

```diff
@@ -108,6 +108,11 @@ public void edit(TaskIndexDto dto, Runnable callback, boolean editedFromTaskGrid
 		editView.addCommitListener(() -> {
 			if (!form.getFieldGroup().isModified()) {
 				TaskDto dto1 = form.getValue();
+				if (!dto1.getAssigneeUser().getUuid().equals(dto.getAssigneeUser().getUuid())) {
```

Comment:
the logic here should be the same as on mobile, you should handle the assignedUser initial null value as well, if it is not null than it should already be set no reason to set it again

Reply:
updated